datajoint/__init__.py

```diff
@@ -75,3 +76,18 @@ class DataJointError(Exception):
 from .schema import Schema as schema
 from .erd import ERD
 from .admin import set_password, kill
+
+
+def create_virtual_module(modulename, dbname):
+    """
+    Creates a python module with the given name from a database name in mysql with datajoint tables.
+    Automatically creates the classed of the appropriate tier in the module.
+
+    :param modulename: desired name of the module
+    :param dbname:     name of the database in mysql
+    :return: the python module
+    """
+    mod = ModuleType(modulename)
+    s = schema(dbname, mod.__dict__)
+    s.spawn_missing_classes()
```

Comment:
Also, would you mind moving this to `datajoint.utils` and importing that in `__init__` instead? `__init__` is starting to get really crowded :)